doc/release/0.5.0-notes.rst

```diff
@@ -6,10 +6,89 @@ PyWavelets 0.5.0 Release Notes
 
 .. contents::
 
+PyWavelets 0.5.0 is the culmination of 1 year of work.  In addition to
+several new features, substantial refactoring of the underlying C and Cython
+code have been made.
+
+This release requires Python 2.6, 2.7 or 3.3-3.5 and NumPy 1.9.1 or greater.
+
+Highlights of this release include:
+
+- 1D continuous wavelet transforms
+- new discrete wavelets added (additional Debauchies and Coiflet wavelets)
+- new 'reflect' extension mode for discrete wavelet transforms
+- faster performance for multilevel stationary wavelet transforms
+- routines to convert multilevel DWT coefficients to and from single array
+- axis support for multilevel DWT
+- substantial refactoring/reorganization of the underlying C and Cython code
+
 
 New features
 ============
 
+1D Continous Wavelet Transforms
+-------------------------------
+A wide range of continous wavelets are now available.  These include the
+following:
+
+- Gaussian wavelets (gaus1...gaus8)
```

Comment:
would be good to put double backticks around all these names, looks better in html docs